raisinbread/RB_files/RB_permissions.sql

```diff
@@ -29,45 +29,44 @@ INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (27,'reliability_swap_candidates','Swap Reliability candidates across all sites',2);
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (28,'instrument_builder','Instrument Builder: Create and Edit instrument forms',2);
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (29,'data_dict_edit','Edit Data Dictionary',2);
-INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (30,'quality_control','Quality Control access',2);
```

Comment:
This arbitrary renumbering of permissions is going to break all the foreign keys to the permissions table in raisinbread.

Reply:
@AlexandraLivadas can you remove the changes to the permission numbers in RB and the schema? They're unnecessary.